Sierra_book.json

```diff
@@ -0,0 +1,96 @@
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "id": "25bbk4m",
+  "type": "Work",
+  "identifiers": 
+    [
+      {
+        "label": "b12141471",
+        "authority": "Sierra"
+      }
+    ],
+  "label": "The Arctic grail: the quest for the North West Passage and the North Pole, 1818-1909",
+  "creators": 
+    [
+      {
+        "label": "Berton, Pierre (1920-2004)",
+        "type": "Agent"
+      }
+    ],
+  "publishers": 
+    [
+      {
+        "label": "Viking",
+        "type": "Agent"
+      }
+    ],
+  "placeOfPublication": 
+     [
+       {
+        "label": "New York",
+        "type": "Place"
+      }
+    ],
+  "publicationDate": 
+    {
+      "label": "1988",
+      "type": "Period"
+    },
+  "edition": "1st American ed.",
+  "isbn": "0670824917"
+  "physicalDescription": "xii, 672 pages.  Other physical details: illustrations, map.  Dimensions: 25 cm",
+  "workType": "Books",
+  "subjects": 
+    [
+     {
+        "label": "Northwest Passage",
+        "authority": "LCSH" 
+      },
+; Comment: Source is identified as LCSH by the second indicator 0.  NB use of MARC $651 indicates this (and the next two subjects) is a geographic subject:
```

Comment:
👎 on YAML

I’d much rather keep these in JSON, so they mirror the API responses as closely as possible.